### PR TITLE
Spring 2019 special primary

### PR DESCRIPTION
**Pull request by nbdavies**
Adding this election: https://elections.wi.gov/elections-voting/results/2019/special-primary-ad64

Followed these steps: https://github.com/davipo/openelections-data-wi#process-for-adding-new-election-data

In the source file, the office was called "SPECIAL PRIMARY REPRESENTATIVE TO THE ASSEMBLY" which is not a standard office name. By removing "Special Primary ", the rest of the office name gets recoded to "State Assembly"

### CODE REVIEW

File: cleaner.py [hunk 1]
```diff
@@ -43,15 +43,15 @@
...
+
     # statewide
     'Governor', 'Lieutenant Governor', 'Attorney General',
     'Secretary of State', 'State Treasurer',
     'State Superintendent of Public Instruction',
-    
+
     # state representatives
     'State Senate', 'State Assembly',
-
```
Comment (davipo) [inline cleaner.py]: There should be no changes to whitespace here (3 lines).

**Review by davipo — CHANGES_REQUESTED** [review]

File: cleaner.py (hunk 1 above)
Comment (nbdavies) [reply cleaner.py]: Oh, I think my editor automatically culled those trailing spaces.

Is it that the whitespace change isn't necessary? Or will removing the trailing spaces have an impact on how the code runs, or violate a style convention?

Comment (davipo) [reply cleaner.py]: Looks like the indents were removed. It's a style convention, shouldn't impact how the code runs. Blank lines should have the same indent as their code block. I think it makes editing easier. The main point is there should not be changes to lines that are not part of the commit.

Comment (davipo) [reply cleaner.py]: There's plenty of debate on whether blank lines should have indentation in Python. For me, when invisible chars are displayed in the editor, using the indents looks more consistent.
https://stackoverflow.com/questions/2727988/python-indentation-in-empty-lines

Comment (davipo) [reply cleaner.py]: Thanks for the fix.

**Review by davipo — APPROVED** [review]

**Pull request merged**